stan/math/prim/scal/fun/inc_beta_ddb.hpp

```diff
@@ -76,8 +76,8 @@ namespace stan {
         summand *= z / k;
 
         if (k > 1e5)
-          throw std::domain_error("inc_beta_ddb did "
-                                  "not converge within 100000 iterations");
+          domain_error("inc_beta_ddb",
+                       "did not converge within 10000 iterations", "", "");
```

Comment:
Yes. Thank you. (I'm really glad you're reviewing code.)

> On Aug 6, 2016, at 5:18 PM, Krzysztof Sakrejda <notifications@github.com> wrote:
> 
> In stan/math/prim/scal/fun/inc_beta_ddb.hpp:
> 
> > @@ -76,8 +76,8 @@ namespace stan {
> >          summand *= z / k;
> >  
> >          if (k > 1e5)
> > -          throw std::domain_error("inc_beta_ddb did "
> > -                                  "not converge within 100000 iterations");
> > +          domain_error("inc_beta_ddb",
> > +                       "did not converge within 10000 iterations", "", "");
> Shouldn't that still be "100000 iterations"?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>